uobtheatre/bookings/models.py

```diff
@@ -446,3 +448,11 @@ def seat_price(self) -> int:
         return self.booking.performance.performance_seat_groups.get(
             seat_group=self.seat_group
         ).price
+
+    def check_in(self):
+        """
+        Check a ticket in
+        """
+        self.checked_in = True
+        self.save()
+        return self
```

Comment:
Why do you return `self` here?

Reply:
I'm guessing this is copied, will nuke it!